submitit/slurm/_sbatch_test_record.txt

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+export SUBMITIT_EXECUTOR=slurm
```

Comment:
This before SBATCH is breaking everything actually, *do not merge*
(I only realized because the partition was not taken into account :s)

Reply:
Solved